app/src/tests/specmath.service.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { TestBed } from '@angular/core/testing';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { Observable } from 'rxjs';
+import { SpecMathService } from '../shared/services/specmath.service';
+import { processFilesMockRequest } from './mocks/mockRequests';
+import { processFilesMockResponseNonConflict, processFilesMockResponseConflict, processFilesMockResponseError } from './mocks/mockResponses';
+import { routes } from '../shared/routes';
+
+describe('SpecMathService', () => {
```

Comment:
I sent you that link for some guidelines we've developed on naming + structuring.  The suggestions I'd make here are a) avoid use of "should", b) group tests for each service call under a separate describe (should there be multiple tests for a service call, anyway).

Reply:
Addressed in commit `d972f64`